pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/models/Sources.scala

```diff
@@ -2,17 +2,34 @@ package uk.ac.wellcome.platform.merger.models
 
 import uk.ac.wellcome.models.work.internal.WorkState.Identified
 import uk.ac.wellcome.models.work.internal._
+import uk.ac.wellcome.platform.merger.rules.WorkPredicates
 
 object Sources {
+  import WorkPredicates._
+
   def findFirstLinkedDigitisedSierraWorkFor(
     target: Work.Visible[Identified],
-    sources: Seq[Work[Identified]]): Option[Work[Identified]] = {
+    sources: Seq[Work[Identified]]): Option[Work[Identified]] =
+    if (physicalSierra(target)
 
-    val digitisedLinkedIds = target.data.mergeCandidates
-      .filter(_.reason.contains("Physical/digitised Sierra work"))
-      .map(_.id.canonicalId)
+        // Audiovisual works are catalogued as multiple bib records, one for the physical
+        // format and another for the digitised version, where available.
+        //
+        // These bibs routinely contain different data, and we can't consider one of
+        // them canonical, unlike other physical/digitised bib pairs.
+        //
+        // Longer term, this may change based if Collections Information change how
+        // they catalogue AV works.
+        //
+        // See https://github.com/wellcomecollection/platform/issues/4876
+        && !isAudiovisual(target)) {
```

Comment:
It's annoying that this has to be in both this method and in the partial rules but it's a bit of a non-trivial refactor to resolve that so agreed that it's probably simpler to keep it like this

Reply:
That said, it's pretty easy to determine you need to do this when making changes – I wrote the obvious test case in PlatformMergerTest, and it failed until I updated both locations. It's not a subtle failure.